test/condition-travis-test.js

```diff
@@ -18,7 +18,7 @@ test.beforeEach(t => {
   delete process.env.TRAVIS;
   delete process.env.TRAVIS_PULL_REQUEST;
   delete process.env.TRAVIS_BRANCH;
-  delete process.env.TRAVIS;
```

Comment:
Why removing this line? It's required to remove the env variable during the test, to avoid testing with the actual `process.env. TRAVIS ` but with the one set in the test itself.

Reply:
It's on line 18 as well. previously double deletion